aws/tf/sra.tf

```diff
@@ -1,89 +1,77 @@
-  read_only_data_bucket              = null  //  Accepts an S3 bucket name ("data-bucket")
-  read_only_external_location_admin  = null  //  This user will become an admin of that external location (e.g. firstname.lastname@company.com)
-
-  // Workspace - operation mode:
-  operation_mode              = "sandbox" // REQUIRED - Accepted values: sandbox, custom, firewall, or isolated. https://github.com/databricks/terraform-databricks-sra/blob/main/aws/tf/README.md#operation-mode
-  compliance_security_profile = false     // REQUIRED - If you are using compliance security profile, please enable this to true. This adds port 2443 (FIPS) as a security group rule.
-
-  // Workspace - AWS non-networking variables:
-  dbfsname                         = join("", [var.resource_prefix, "-", var.region, "-", "dbfsroot"])
-  cmk_admin_arn                    = null  // If not provided, the root user of the AWS account is used
-  enable_cluster_boolean           = false // WARNING: Clusters will spin-up Databricks clusters and AWS EC2 instances
-  enable_admin_configs             = false // WARNING: The workspace_conf resource is evolving API that may change from provider to provider. Please review the in-resource documentation (admin_configuration.tf) before enabling.
-  workspace_service_principal_name = "sra-example-sp"
 
-  // Workspace - networking variables (optional if using custom operation mode):
-  vpc_cidr_range           = "10.0.0.0/18"
-  private_subnets_cidr     = ["10.0.16.0/22", "10.0.24.0/22"]
-  privatelink_subnets_cidr = ["10.0.32.0/26", "10.0.32.64/26"]
-  public_subnets_cidr      = ["10.0.32.128/26", "10.0.32.192/26"]
-  availability_zones       = [data.aws_availability_zones.available.names[0], data.aws_availability_zones.available.names[1]]
-  sg_egress_ports          = [443, 3306, 6666, 8443, 8444, 8445, 8446, 8447, 8448, 8449, 8450, 8451]
-  sg_ingress_protocol      = ["tcp", "udp"]
-  sg_egress_protocol       = ["tcp", "udp"]
-  relay_vpce_service       = var.scc_relay[var.region]
-  workspace_vpce_service   = var.workspace[var.region]
+  // Required Variables:
+  workspace_catalog_admin                = null             // Workspace catalog admin email.
+  user_workspace_admin                   = null             // Workspace admin user email.
+  operation_mode                         = "sandbox"        // Operation mode (sandbox, custom, firewall, isolated).
+  workspace_admin_service_principal_name = "sra-example-sp" // Creates an example admin SP for automation use cases.
+  metastore_exists                       = false            // If a regional metastore exists set to true.
+
+  // AWS Specific Variables:
+  cmk_admin_arn                            = null // CMK admin ARN, defaults to the AWS account root user.
+  vpc_cidr_range                           = "10.0.0.0/18"
```

Comment:
Since this file is filled with comments, I can say that I have had confusion when I send over VPC CIDR prepopulated, might be worth adding a comment here just as //Example CIDR, must update private_subnets_cidr, public_subnets_cidr, privatelink_subnets_cidr, and/or firewall_subnets_cidr if changed

Reply:
like the VPC CIDR is 172.* then the subnets remain the same?

Reply:
Yeah, the following CIDR ranges would be invalid. Just something like // If updated, must update below subnets